wp-stash.php

```diff
@@ -12,21 +12,9 @@
 
 namespace Inpsyde\WpStash;
 
-if (! class_exists(WpStash::class)) {
-    if (is_readable(__DIR__ . '/vendor/autoload.php')) {
-        /** @noinspection PhpIncludeInspection */
-        require_once __DIR__ . '/vendor/autoload.php';
-    } else {
-        foreach (['admin_notices', 'network_admin_notices'] as $hook) {
-            add_action($hook, function () {
-
-                $message = 'Could not find a working autoloader for ' . __NAMESPACE__;
-                $class = 'notice notice-error';
-                printf('<div class="%1$s"><p>%2$s</p></div>', esc_attr($class), esc_html($message));
-            });
-        }
-
-        return;
-    }
+if (!class_exists(WpStash::class) && file_exists(__DIR__.'/vendor/autoload.php')) {
+    /** @noinspection PhpIncludeInspection */
+    require_once __DIR__.'/vendor/autoload.php';
```

Comment:
Is there a reason for the removal of the admin notice?

And why the move from `is_readable` to `file_exists`? The former implies the latter and has an additional check for access permissions.